SECURITY.md

```diff
@@ -1,17 +1,116 @@
+   communication with the reporter throughout the process, providing
+   updates on the status of the vulnerability and any steps taken to
+   remediate it. In particular this means that the reporter will be asked to review any proposed fix and to advise on the timing for public disclosure.
+
+5. **Fixing Issue**: The team agrees on the fix, the announcement, and the release schedule with the reporter. If the reporter is not responsive in a reasonable time frame this should not block the team from moving to the next steps particularly in the face of a high impact or high severity issue.
+
+   a. **Mitigation**: Depending on the severity and criticity of the issue, the team can decide to disclose the issue publicly in the absence of a fix _if and only if_ a clear, simple, and effective mitigation plan is defined. This _must_ include instructions for users and operators of the software, and a time horizon at which the issue will be properly fixed (eg. version number).
+
+   b. **Fix**: When a fix is available and approved, it should be merged and made available as quickly as possible:
+
+      * All commits to the private repository are squashed into a single commit whose description _should not_ make any reference it relates to a security vulnerability
+      * A new Pull Request is created with this single commit
+      * This PR's review and merging is expedited as all the work as already been done
+
+6. **Release**: The team creates and publish a release that includes the fix
+
+7. **Announcement**: Concommitant to the release annoucement, the team announces the security vulnerability by making the GitHub issue public. This is the first point that any information regarding the vulnerability is made public.
+
+    a. **Credit**: The team publicly acknowledges the contributions of the
+       reporter once the vulnerability is resolved, subject to the
+       reporter's preferences for attribution.
+
+7. **Disagreements**: In case of disagreements with the reporter on the fix, mitigation, timing, or announcement, the team has the final say.
+
+## Responsible Disclosure
+
+We kindly request that reporters adhere to responsible disclosure
+practices, which include:
+
+- **Do not disclose the vulnerability publicly**: Please refrain from
+  posting details of the vulnerability on public forums or social
+  media until it has been resolved.
+- **Do not exploit the vulnerability**: Do not attempt to exploit the
+  vulnerability to cause harm or gain unauthorized access to systems.
+- **Work with us**: Allow us a reasonable amount of time to
+  investigate and address the vulnerability before publicly disclosing
+  any details.
+
+## Legal Protections
+
+The Hydra team is committed to providing legal protections to
+reporters who act in good faith and follow responsible disclosure
+practices. We will not pursue legal action against individuals who
+report security vulnerabilities to us.
```

Comment:
```suggestion
We will not pursue legal action against individuals who
report security vulnerabilities to us.
```

When we say that _the Hydra team is committed to providing legal protections to reporters who act in good faith and follow responsible disclosure practices_, it's not clear to me what we mean by _legal protection_. Are we paying for their layer? Or is it a standard statement that has a clear meaning and value in some jurisdiction?

In doubt, I would remove this part of the sentence.